### ISSUE TITLE
Combo box has no associated label

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11253]
## Expected Behavior

Combo box should have a defined associated label so screen readers can detect it and announce to the users.

## Actual Behavior

1. Navigate to `Home` on the left navigation. Make sure `Components` is selected on menu below the main heading.
2. Select `all` filter and in the table navigate to any application under name column with application live view plugin available (see screenshot below).
3. In the next page that opens up for selected appplication, please navigate to `Runtime resources` tab.
4. Navigate to `Resources` table and select the name of the application. A new page opens up.
5. Navigate to `Pods` table and select a link under name column.
6. In the new page that opens up, navigate to `Live view` section.
7. Select `Metrics` item from the menu item.
8. Verify whether screen reader announces the label or not when selecting the menuitem in the combo box. 

![image](https://user-images.githubusercontent.com/54085737/166465803-abbad94c-15a7-40a6-826e-ab8d5f2b68ea.png)

## Context

Combo box has no label and screen reader announces "without label menu button submenu".

User Impact: Without the name of the combo box, it is difficult for screen reader users to know about the functionality of it. 
Thereby, they might not use the combo box.

A11y Conformance Level: A

WCAG Success Criteria: 1.3.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4

**Comment by freben** [comment]
@claudiahub Same here - is this really an open source plugin?

**Comment by claudiahub** [comment]
@freben same here, this is an internal plugin but the components in this view are located on @backstage/core-components so I guess we should address there this accessibility issue.

**Comment by claudiahub** [comment]
@Rugvip sure, I forgot to add the ''closes #issue''. thanks for pointing it out and closing the issue!
